### DISCUSSION TITLE
Ability to search shortcuts via hotkey & highlight the matching shortcuts/keybinds

### DISCUSSION THREAD

**Discussion by iotaOmicron** [Anze/KeyCluCask#31] [Ideas]
It would be great if there was a way to "search" through the panel via hotkey. KeyCue uses ⎈⌘F I think as default, and it automatically changes to ⌥⌘F or something else if it detects a conflict within the app it's surveying. So if I hit the hotkey, a bar could pop up at the bottom displaying the current input, and if, for example, I held G, all the keyboard shortcuts in the app it's surveying that include "G" would have their title changed to the KeyClu accent color (blue/purple?), or every other shortcut that doesn't include "G" could dim, but I prefer the former. It would revert on lifting the keys.

&nbsp;

note, this would require an easier way to make the "nonpersistent" full-screen view easier to keep up. Currently it seems calling via shell is the only consistent way of doing it other than bringing it up, right-clicking, and then lifting the cmd key (which would normally dismiss it). I was going to make a separate thread about this, but I think the app would benefit from an adaptive panel window size or better column sorting so you don't have to scroll down to see things. That in my opinion is the only downside I can see compared to KeyCue, otherwise you've surpassed it, and so quickly too. It does have the "reprogramming by holding" feature, but the right-click to edit with CustomShortcuts thing is almost as good

**Comment by Anze** [comment]
Thanks for bringing up these topics.

Yeah, someone already asked about this on Reddit. Back then, I thought it was easier to search using ⌘+⇧+/. However, now that I'm working on the feature for manually registering shortcuts/hotkeys, I see its real usefulness.

Regarding the persistent panel, I also have plans to enhance it with more layout possibilities.
